### PR TITLE
show where KDIR is failing to help troubleshooting

### PR DESCRIPTION
**Pull request by kintarowins**
This gives better indication of what's wrong and how (the path) in order to fix the issue

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -88,7 +88,8 @@ doc:
 .PHONY: check-kdir
 check-kdir:
 	@if ! test -e $(KDIR)/Makefile ; then \
-		echo "    SORRY, kernel makefile not found." ;\
+		echo "    SORRY, kernel makefile not found in KDIR:" ;\
```
Comment (rck) [inline Makefile:91]: why not just
`echo "    SORRY, kernel makefile not found in '$(KDIR)'" ;\`?

**Review by rck — CHANGES_REQUESTED** [review]
hm, yes, no, don't know. fixin/improving things in out-of-tree 8.4 is IMO pretty pointless. All of 8.4 should be in upstream kernels for a long time. And out-of-tree people should use DRBD 9.2

**Comment by rck** [comment]
I reopen it because the bot closed it.

**Comment by rck** [comment]
also: I don't see a scenario where this would even land in some kind of release, as as I said, 8.4 out-of-tree is basically EOL

**Comment by kintarowins** [comment]
I didn't realize that. Thanks. Reopened in https://github.com/LINBIT/drbd/pull/48

**Pull request closed without merging**